tools/ci_build/build.py

```diff
@@ -622,6 +624,10 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs, enab
                 if onnxml_test:
                     run_subprocess([sys.executable, 'onnxruntime_test_python_keras.py'], cwd=cwd, dll_path=dll_path)
 
+def run_winml_tests():
+    print('WinML tests not implemented')
+    # raise NotImplementedError()
```

Comment:
will this change? isn't this the place to run the tests?

Reply:
Currently there's no build.py flag to run only WinML tests (you can run it manually by calling the binary directly), I'll either remove this or add more options to build.py.

I'll probably remove and if an user needs only one test project they can call it directly.